app/channels/application_cable/connection.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
please remove these comments from all files

Reply:
i think i got all now, thank you